src/changes-backwards-compatible.apib

```diff
@@ -6,6 +6,8 @@ We list all backwards-compatible additions here. These are currently available i
 
 #### Latest
 
+- We removed all `Work Orders` documentation. 
```

Comment:
PP : please don't mention this in the changelog.

The reason we would unpublish these docs is to remove focus from this, since the group of endpoints is not complete. No need to stress that out